todo_app/app.py

```diff
@@ -1,35 +1,40 @@
-import flask
+def create_app():
+    import flask
 
-from todo_app.config import FlaskConfig, TrelloConfig
-from todo_app.trello import Trello
+    from todo_app.config.flask import FlaskConfig
+    from todo_app.config.trello import TrelloConfig
+    from todo_app.models import ViewModel
+    from todo_app.trello import Trello
```

Comment:
This is a valid solution for delaying the imports until the tests have been able to configure the environment, but note that it's standard to keep imports at the top of the file as much as we can. An alternative might be to switch to using object variables rather than class variables for your config; e.g. always instantiating a relevant config object, and setting the variables in the constructor, for the Flask_Config something like:
```python

class FlaskConfig:
    """Base configuration variables."""
    def __init__(self):
        self.SECRET_KEY = os.environ.get('SECRET_KEY')
        if not self.SECRET_KEY:
            raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
```